docs/content/guides/4.customization/2.app-router/2.ui-customizations.md

```diff
@@ -0,0 +1,380 @@
+
+In this chapter, you will:
+
+::list{type="success"}
+- change the default logo to your custom one
+- adjust the theme colors to suit the new logo
+- add a pre-header with i18n
+- customize the look of product cards on the product listing page
+- add facet/filter search feature
+::
+
+In the end, your application will look something like this:
+
+![Customizations](./images/customizations.webp)
+
+## Changing the logo
+
+The first step to making any changes is to identify what component is actually responsible for certain elements.
+
+In this example, we need to find and modify the component that contains the logo. You can either:
+
+1. Drill through the Next.js application starting from `apps/storefront-unified-nextjs/app/[locale]/(cms)/[[...slug]]/page.tsx`, which represents
+the homepage (actually all CMS pages). The layout for this page is located in `apps/storefront-unified-nextjs/app/[locale]/(cms)/layout.tsx`, which re-exports
+`BaseDefaultLayout`. In that layout you can find that `Navbar` component is responsible for rendering the navbar. Within that component we have `NavbarTop` component
+which renders the logo.
+2. Or you can use [React Developer Tools](https://react.dev/learn/react-developer-tools) to localize the component visually:
+![React Dev Tools](./images/react-dev-tools.webp)
+
+Doing one of these, you'll find that the logo is located in `apps/storefront-unified-nextjs/components/navigations/navbar-top.tsx` component.
+
+Now, we have to change that component and replace `SfIconAlokaiFull` with your custom image. For this example, let's use [LogoIpsum](https://logoipsum.com/) to generate a sample logo. Download a sample logo and place it in `apps/storefront-unified-nextjs/public/images` folder.
+
+```html
+<!-- [!code --:2]-->
+<SfIconAlokaiFull data-testid="logo" className="h-full w-auto" />
+<!-- [!code ++:2]-->
+<Image src="/images/<logo-file-name>" width={100} height={50} unoptimized alt="logo" />
+```
+
+The result should look like this:
+
+![Logo](./images/logo-on-green.webp)
+
+However, this logo doesn't look good on the storefront's default green background. We can change that by removing the `filled` property from `NavbarTop` component in `Navbar` component. This will make the navbar transparent.
+
+```diff[apps/storefront-unified-nextjs/app/[locale]/(default)/components/navbar.tsx]
+- <NavbarTop filled>
++ <NavbarTop>
+```
+
+The logo should look better now. However, buttons in the navbar have become invisible. That's because they are white.
+
+::tip 
+#title
+An Extra Challenge
+#default
+Within the `NavbarTop` component, find which tailwind class is responsible for making the buttons white and remove it.
+::
+
+## Adjusting theme colors
```

Comment:
Probably also a good time to link to https://docs.storefrontui.io/v2/customization/theming

Reply:
It's linked down below ;)